cmd/cycloid/stacks/get-config.go

```diff
@@ -71,13 +71,19 @@ func getConfig(cmd *cobra.Command, args []string) error {
 	}
 
 	ref, err := cmd.Flags().GetString("ref")
+	if err != nil {
+		return errors.Wrap(err, "unable to get ref flag")
+	}
```

Comment:
This check should not exist, if you check for err here you block the usage of the argument for the command.